include/openssl/byteorder.h

```diff
@@ -21,6 +21,7 @@
  */
 
 # if defined(_MSC_VER) && _MSC_VER>=1300
+#  include <intrin.h>
```

Comment:
The manpage says to use <stdlib.h>

Reply:
Updated the PR to include `stdlib.h` instead.